EMS/core-bundle/src/Controller/ContentManagement/CrudController.php

```diff
@@ -45,20 +45,20 @@ public function createAction(?string $ouuid, string $name, Request $request): Re
                 $this->dataService->refresh($newRevision->giveContentType()->giveEnvironment());
             }
         } catch (\Exception $e) {
-            if (($e instanceof NotFoundHttpException) or ($e instanceof BadRequestHttpException)) {
```

Comment:
WTF "or". Good catch

Reply:
We have more :) but will not do in this PR